roles/ovsconfig/templates/ifcfg-sub-interface.j2

```diff
@@ -0,0 +1,7 @@
+DEVICE=bond0.{{item['vlanid']}}
+ONBOOT=yes
+BOOTPROTO=none
+TYPE=Vlan
+VLAN=yes
+IPADDR={{item['ip']}}
+NETMASK={{item['mask']}}
```

Comment:
This should be fine assuming it's not a management network which would need to the extra items below added.

GATEWAY=192.0.2.1
DNS1=192.0.2.100
DNS2=192.0.2.200